tree.ml

```diff
@@ -353,6 +356,7 @@ let mk_eval num_observations =
         | `Leaf g ->  gamma.(i) <- g
         | `OrdinalNode _
         | `CategoricalNode _ -> assert false
+        | `LinearNode _ -> assert false
```

Comment:
How do I evaluate LinearNodes in this context? I need the actual (float) value of the feature.